README.md

```diff
@@ -22,6 +22,7 @@ Contributions are always welcome!
 - [Capacity Planning](#capacity-planning)
 - [Service Level Agreement](#service-level-agreement)
 - [Performance](#performance)
+- [Release](#release)
```

Comment:
A comment on the wording: this should be "Releasing" instead of "Release", or could be changed to something like "Release Management".

Reply:
Scratch the "Release Management" suggestion, since it seems you have a link in the section to differentiate between that and "Release Engineering".